app/vmselect/promql/aggr_incremental.go

```diff
@@ -105,7 +104,7 @@ func (iafc *incrementalAggrFuncContext) updateTimeseries(tsOrig *timeseries, wor
 	removeGroupTags(&ts.MetricName, &iafc.ae.Modifier)
 	bb := bbPool.Get()
 	bb.B = marshalMetricNameSorted(bb.B[:0], &ts.MetricName)
-	k := bytesutil.InternBytes(bb.B)
+	k := string(bb.B)
```

Comment:
I don't think it is clear to me now when I should use `k := string(bb.B)` and when just `iac := m[string(bb.B)]`. Or go compiler is smart enough to optimize first case?